### ISSUE TITLE
Asari Viz Should Report Parameters Used for Analysis

### ISSUE THREAD

**Issue by jmmitc06** [shuzhao-li-lab/asari#47]
Hi, 

Today while I was comparing results from multiple asari runs, it would have been helpful to have a tab where I could see the parameters that were used to run asari.

**Comment by YuanyeChi** [comment]
Yes sure. Maybe another block on top of data summarry just picking information from project.json.

**Comment by jmmitc06** [comment]
Done in V1.14. Convert JSON parameters to pd.DataFrame then render as scrollable table. Not the prettiest, but functional.
